Corona-Warn-App/src/main/res/values-de/strings.xml

```diff
@@ -1280,13 +1280,34 @@
 
     <!-- Incidence statistics card -->
     <!-- XHED: Title for incident statistics card -->
-    <string name="statistics_card_incidence_title">7-Tage-Inzidenz</string>
-
-    <!-- XTXT: Label in statistics card that is displayed above the incidence value-->
-    <string name="statistics_card_incidence_value_label">Bis gestern</string>
-
+    <string name="statistics_card_incidence_title">"7-Tage-Inzidenz"</string>
     <!-- XTXT: Description in statistics card that is displayed below the incidence value-->
-    <string name="statistics_card_incidence_value_description">bestätigte Neuinfektionen je 100.000 Einwohner</string>
+    <string name="statistics_card_incidence_value_description">"bestätigte Neuinfektionen je 100.000 Einwohner"</string>
```

Comment:
```suggestion
    <string name="statistics_card_incidence_value_description">"Bestätigte Neuinfektionen je 100.000 Einwohner"</string>
```

Reply:
FYI: I asked for clarification as it mismatches Figma

Reply:
Sabine agreed to keep the small caps